easybuild/easyconfigs/r/ReFrame/ReFrame-3.3.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'PythonBundle'
+
+name = 'ReFrame'
+version = '3.3'
+
+homepage = 'https://github.com/eth-cscs/reframe'
+description = '''ReFrame is a framework for writing regression tests for HPC systems.'''
+
+toolchain = SYSTEM
+
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+req_py_majver = 3
+req_py_minver = 6
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+use_pip = True
+
+exts_list = [
+    ('pip', '20.2.4', {
+        'use_pip': False,
+        'checksums': ['85c99a857ea0fb0aedf23833d9be5c40cf253fe24443f0829c7b472e23c364a1'],
+    }),
+
+    # Add a setuptools version supporting the `find_namespace_packages` function
+    ('setuptools', '50.3.2', {
+        'source_tmpl': 'setuptools-%(version)s.zip',
+        'checksums': ['ed0519d27a243843b05d82a5e9d01b0b083d9934eaa3d02779a23da18077bd3c'],
+    }),
+    ('reframe', version, {
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/eth-cscs/reframe/archive/'],
+        'checksums': ['9da150a051e9fa4ffea1361f30e8593261e7f6ebc71ec91ed32143539f871ad7'],
+    }),
+]
+
+postinstallcmds = [
+    "cd %(builddir)s/reframe/reframe-%(version)s && ./bootstrap.sh +pygelf && "
```

Comment:
@teojgo Please make this two separate commands (easier to debug if something goes wrong), and add a comment to explain what the bootstrap script does, and why it's used).